providers/google_test.go

```diff
@@ -102,7 +102,8 @@ func TestGoogleProviderGetEmailAddress(t *testing.T) {
 	p.RedeemURL, server = newRedeemServer(body)
 	defer server.Close()
 
-	session, err := p.Redeem("http://redirect/", "code1234")
+	ctx := context.TODO()
```

Comment:
Please do the others in this file too
```suggestion
	ctx := context.Background()
```